content/en/docs/setup/install/virtual-machine/index.md

```diff
@@ -0,0 +1,188 @@
+---
+title: Installing Istio for virtual machine connectivity
+description: Install an Istio mesh such that a virtual machine can connect to it and configure the virtual machine.
+weight: 40
+keywords:
+- kubernetes
+- virtual-machine
+- gateways
+- vms
+---
+
+Follow this guide to deploy a [multicluster service mesh](/docs/ops/deployment/deployment-models/#multiple-clusters) spanning one more more clusters and then connect a virtual machine to it.
+
+{{< warning >}}
+This guide relies heavily on the
+[stable plugin ca certificate](/docs/tasks/security/cert-management/plugin-ca-cert/) feature. Istio
+does not offer an industry standard recommendation for certificate management. Please consult your
+information security team when using any plugin certificate.  The determination of your company
+policies for certificate management is the choice of your company.
+{{< /warning >}}
+
+{{< tip >}}
+This guide is tested and validated. The Istio authors feel this guide is suitable for experimentation
+but not production. Like all alpha features, this guide is subject to change.
+{{< /tip >}}
+
+## Prerequisites
+
+1. [Download the Istio release](/docs/setup/getting-started/#download)
+1. Perform any necessary [platform-specific setup](/docs/setup/platform-setup/)
+1. Check the requirements [for Pods and Services](/docs/ops/deployment/requirements/)
+1. Virtual machines must have IP connectivity to the ingress gateway in the connecting mesh, and optionally every pod in the mesh via L3 networking if enhanced performance is desired.
+
+## Installing the Istio control plane
+
+1. Set the name of your cluster as an environment variable.
+
+    {{< text bash >}}
+    $ CLUSTER_NAME="<the name of your cluster>"
+    {{< /text >}}
+
+1. Set the name of your desired namespace as an environment variable.
+
+    {{< text bash >}}
+    $ SERVICE_NAMESPACE="<the name of your service namespace>"
+    {{< /text >}}
+
+1. Create a working directory for files generated in this install guide.
+
+    {{< text bash >}}
+    $ mkdir -p "${HOME}"/"${CLUSTER_NAME}"
```

Comment:
No reason to create both directories if using the -p. Just use the second line.

Reply:
good optimization - thanks.